src/LevelEditor.cpp

```diff
@@ -1714,44 +1718,59 @@ std::string MoveGameObjectCommand::describe() {
 std::string AddRemoveGameObjectCommand::describe() {
 	if (objects.size() == 1) {
 		Scenery *scenery = dynamic_cast<Scenery*>(objects[0]);
-		return tfm::format(isAdd ? _("Add %s") : _("Remove %s"), scenery ? (scenery->sceneryName_.empty() ? _("Custom scenery block")
+		return tfm::format(isAdd ? _("Add %s") : _("Remove %s"), scenery ? (scenery->sceneryName_.empty() ?
+			/// TRANSLATORS: Context: Add/Remove ...

```

Comment:
I think this comment is also wrong, there is actually a scenery block named "Custom scenery block". Maybe you should added a comment two lines above?

Reply:
See my comment above - this it the context for this occurrence of the string.